app/helpers/application_helper/button/condition.rb

```diff
@@ -0,0 +1,19 @@
+class ApplicationHelper::Button::Condition < ApplicationHelper::Button::ReadOnly
```

Comment:
@jzigmund The `role_allows_feature?` methods in added button classes seems to be exactly the same.

Can't this class inherit from `ApplicationHelper::Button::ConditionPolicy`, so you don't need to redefine `role_allows_feature?` method here again?